app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -250,4 +238,50 @@ public function store(Request $request, $id)
         return redirect()->to('account/accept')->with('success', $return_msg);
 
     }
+    /**
+     * returns the checkout Acceptance agreement to a pending state for user to resign the agreement.
+     *
+     * @param  int  $id
+     * @return redirect
+     */
+
+    public function resignAcceptance($id){
+
+        $acceptance = CheckoutAcceptance::where('checkoutable_id', $id)
+                                        ->where('checkoutable_type','App\Models\Asset')
+                                        ->latest('created_at')
+                                        ->get()
+                                        ->first();
```

Comment:
Nitpick: I think you should be be able to drop the `get()` and call `first()` directly.